plugin_dockerbuild/config.json-template

```diff
@@ -21,7 +21,7 @@
 			"Options": ["rbind"]
 		},
 		{
-			"Description" : "Location to look for config file (/etc/docker-volume-vsphere.conf)",
+			"Description" : "Location to look for config file (/etc/BINARY.conf)",
```

Comment:
I strongly recommend using some pa1ttern like %BINARY% to separate from comments etc.
Or better  , try using golang template module